src/util/selectorAtCursor.js

```diff
@@ -0,0 +1,27 @@
+import postcss from 'postcss';
+
+export function selectorAtCursor(source, cursor, language) {
+  let highlighterSelector = '';
+  if (language === 'css') {
+    try {
+      const root1 = postcss.parse(source);
+      root1.walkRules((rule) => {
+        const ruleStartRow = rule.source.start.line;
+        const ruleStartCol = rule.source.start.column;
+        const ruleEndRow = rule.source.end.line;
+        const ruleEndCol = rule.source.end.column;
+        const cursorRow = cursor.row + 1;
+        const cursorCol = cursor.column + 1;
+        if ((cursorRow > ruleStartRow && cursorRow < ruleEndRow) ||
+          (cursorRow === ruleStartRow && cursorCol >= ruleStartCol) ||
+          (cursorRow === ruleEndRow && cursorCol <= ruleEndCol)) {
+          highlighterSelector = rule.selector;
```

Comment:
Per the [API docs](http://api.postcss.org/global.html#childIterator) we can `return false` here to break iteration if we’ve found a match, which is a mild gain for runtime efficiency.